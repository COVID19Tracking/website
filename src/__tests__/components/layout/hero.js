import React from 'react'
import renderer from 'react-test-renderer'
import Hero from '../../../components/layout/hero'

describe('Components : Layout : Hero', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Hero />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
