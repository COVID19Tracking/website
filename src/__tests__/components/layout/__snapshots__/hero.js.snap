// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Layout : Hero renders correctly 1`] = `
<div
  className="container false container"
>
  <div
    className="layout--content-full"
  >
    <div
      className="layout--content-primary"
    >
      <div
        className="hero hero"
      >
        <p
          className="header"
        >
          The COVID Tracking Project collects and publishes the most complete testing data available for US states and territories.
        </p>
        <div
          className="ctas"
        >
          <a
            className="button"
            href="/data"
          >
            Get the Data
          </a>
          <span>
            or
          </span>
          <a
            href="/about-data"
          >
            Learn About the Data
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
